spec.bs

```diff
@@ -97,45 +107,42 @@ A <dfn>Data Point</dfn> is a single point of a [=Time Series=] containing an ISO
-- `tss:timePath`: the path to use for expanding the `time` property in `tss:points`.
-- `tss:valuePath`: the path to use for expanding the `value` property in `tss:points`. 
+- `tss:context`: JSON-LD context describing the JSON array of `tss:points`.
 
-Issue: Discuss whether these properties are required or optional. E.g., a publisher might decide to do a lossy conversion for their goal, and not include valuePath, pointType and timePath. However, we can still analyze and visualize the data without that information.
+Issue: Discuss whether these properties are required or optional. E.g., a publisher might decide to do a lossy conversion for their goal, and not include a JSON-LD context or pointType. However, we can still analyze and visualize the data without that information.
 
 ## Data Points ## {#points}
 
-`tss:points` MUST have a `cdt:List` as datatype. Each [=Data Point=] itself MUST be a `cdt:Map` consisting
-of 2 required properties and 1 optional property:
+`tss:points` MUST be a JSON array with JSON objects (`rdf:JSON` as datatype).
+Each [=Data Point=] itself MUST be a JSON object consisting of 2 required properties and 1 optional property:
   - `time`: the timestamp of the data point using an `xsd:dateTime`.
   - `value`: the value of the data point with corresponding datatype.
   - `id`: the data point identifier is optionally. When set, this MUST be a named node.
 
 <div class="example" id="ex_points">
 ```turtle
-[
-    { "time": "2026-01-01T06:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.4"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/0" },
-    { "time": "2026-01-01T06:59:59Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.2"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/1" },
-    { "time": "2026-01-01T08:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "5.2"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/2" },
-    { "time": "2026-01-01T09:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime>, "value": "6.1"^^<http://www.w3.org/2001/XMLSchema#double>, "id": "https://example.org/3" },
-]
+"""[
+    { "time": "2026-01-01T06:00:00Z", "value": "5.4", "id": "https://example.org/0" },
+    { "time": "2026-01-01T06:59:59Z", "value": "5.2", "id": "https://example.org/1" },
+    { "time": "2026-01-01T08:00:00Z", "value": "5.2", "id": "https://example.org/2" },
+    { "time": "2026-01-01T09:00:00Z", "value": "6.1", "id": "https://example.org/3" },
+]"""^^rdf:JSON;
 ```
 </div>
 
 ## Expanding data points ## {#expanding}
 
-When `tss:timePath` and `tss:valuePath` are set (mind this is not required), a [=Snippet=] can be expanded to a verbose RDF representation, for example using its original vocabulary.
+When the optional JSON-LD context is provided through `tss:context`, a [=Snippet=] can be expanded to a verbose RDF representation, for example using its original vocabulary.
```

Comment:
I think we need to specify the default context here. I would suggest something like this:

```json
{
 "@context": {
    "id": "@id",
   "time: {
       "@id": "https://w3id.org/tss#time"
       "@type":"xsd:dateTime"
    },
    "value": "https://w3id.org/tss#value" 
 } 
}
```

And simply specify that we use the JSON-LD algorithm to turn this then into quads.